### DISCUSSION TITLE
Understanding metrics API descriptions

### DISCUSSION THREAD

**Discussion by monil-patel** [github-early-access/copilot-custom-models-beta#15] [Q&A]
Could someone clarify the distinction between a "suggestion" and a "line of code"? Does "suggestions" represent non code responses like comments? or does a line mean a certain length? What if copilot is suggesting the next function name where would that be captured? 

I found these on the [docs](https://docs.github.com/en/rest/copilot/copilot-usage?apiVersion=2022-11-28) but it wasnt clicking for me personally yet. 

```json
"suggestions_count": {
  "type": "integer",
  "description": "The number of Copilot suggestions shown to users in the editor specified during the day specified."
},
"acceptances_count": {
  "type": "integer",
  "description": "The number of Copilot suggestions accepted by users in the editor specified during the day specified."
},
"lines_suggested": {
  "type": "integer",
  "description": "The number of lines of code suggested by Copilot in the editor specified during the day specified."
},
"lines_accepted": {
  "type": "integer",
  "description": "The number of lines of code accepted by users in the editor specified during the day specified."
},
```

**Comment by justincampbell** [comment]
@monil-patel Copilot can offer completion suggestions that are a single line (completing the current line), or multiple lines (for instance, an entire function implementation). Suggestions and acceptances are based on any suggestion length, and the lines suggested/accepted are individual lines. I hope this helps! Let us know if you have any further questions.
